starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,43 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(arr1){
+  var longestWord = '';
+  if (longestWord = ''){
+    longestWord = undefined;
+  }
+  else{
+  for (var i =0; i < arr1.length; i++){
+    if (longestWord.length < arr1[i].length){
+      longestWord = arr1[i];
+    }
+  }}
+  return longestWord;
+}
 
+// Calculating a Sum
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray([]){
```

Comment:
You would not pass an empty array. This is the correct method, so you can compare:
```js
// Calculating a Sum
function sumArray(array) {
  var sum = 0;
  for (var x = 0; x < array.length; x++) {
    sum += array[x];
  }
  return sum;
}

var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
var total = sumArray(numbers);
console.log(total);
```